scripts/pre_commit_linter.py

```diff
@@ -220,6 +220,13 @@
         'excluded_files': (),
         'excluded_dirs': ('core/tests/')
     },
+    {
+        'regexp': r'\/\/\sTODO[^\(]*[^\)]$',
```

Comment:
A couple of points here:
1. I think it would be better if we put the regexp in `BAD_PATTERNS` since this needs to be checked for all files, irrespective of the type.
2. Let's enforce the `TODO(@username):` pattern instead of `TODO(username):`.

Reply:
Gotcha, will enforce the @ sign! 

I actually thought about putting the regexp in `BAD_PATTERNS`, but all of those patterns are checked [here](/oppia/oppia/blob/develop/scripts/pre_commit_linter.py#L2166) as just seeing if the pattern exists in the file, not with a regular expression. I would be happy to find a pattern that works for that method if you prefer! Something like `TODO: ` as a bad pattern

Reply:
@teddymarchildon, I see. Can we set up a `BAD_PATTERNS_REGEXP` for this (since we have bad patterns regexp for different file types)? We can then use the same mechanism. Thanks!

Reply:
Hi @teddymarchildon, have you looked into this? Thanks!

Reply:
Ah I must've missed that comment, working on it now!

Reply:
@apb7 Updated! Much cleaner - thanks for the heads up.